### PR TITLE
Retries 7x

### PR DESCRIPTION
**Pull request by andrewvc**
Moved #647 to here since I accidentally closed the branch . which is apparently fatal.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ env:
  - INTEGRATION=true ES_VERSION=1.7.6 TEST_DEBUG=true
  - INTEGRATION=true ES_VERSION=2.4.4 TEST_DEBUG=true
  - INTEGRATION=true ES_VERSION=5.5.0 TEST_DEBUG=true
- - INTEGRATION=true ES_VERSION=master TEST_DEBUG=true
+ - INTEGRATION=true ES_VERSION=6.x TEST_DEBUG=true
```
Comment (suyograo) [inline .travis.yml]: @andrewvc is this change intentional?

Comment (andrewvc) [inline .travis.yml]: Yes. I don't think the 7.x branch of this plugin is intended to be tested against Logstash 7.0, which I assume will only work with the 8.x branch.

Comment (andrewvc) [inline .travis.yml]: There's no specific breakage now, but I think this still makes sense

Comment (suyograo) [inline .travis.yml]: @andrewvc this will still fail because there is no `6.x` version. It needs to be `6.0.0-beta1` I think
